core/templates/dev/head/domain/collection/ReadOnlyCollectionBackendApiService.js

```diff
@@ -21,7 +21,7 @@
 // separate CollectionDataService implementation which returns a local copy of
 // the collection instead. This file should not be included on the page in that
 // scenario.
-oppia.factory('CollectionBackendApiService', [
+oppia.factory('ReadOnlyCollectionBackendApiService', [
```

Comment:
Can you please make sure the ``fileoverview`` specifically mentions this provides read-only collections?